srt_test.go

```diff
@@ -127,3 +127,24 @@ func TestSRTStyled(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, string(c), w.String())
 }
+
+func TestSRTHTMLEntity(t *testing.T) {
```

Comment:
Could you :
- move this test to `subtitles_test.go`
- rename it to `TestHTMLEntities`
- rename `example-in-html-entity.srt` to `example-in-html-entities.srt` and duplicate it to `example-out-html-entities.srt`
- rename `example-out-html-entity.vtt` to `example-out-html-entities.vtt` and duplicate it to `example-in-html-entities.vtt`
- make sure all those files have `&amp;`, `&lt;` and `&nbsp;` in them
- in the test do the following
  - open `in.srt`
  - assert lines like [here](https://github.com/asticode/go-astisub/blob/master/webvtt_test.go#L161)
  - assert write to srt with `out.srt` file and write to webvtt with `out.vtt`
  - open `in.vtt`
  - assert lines like [here](https://github.com/asticode/go-astisub/blob/master/webvtt_test.go#L161)
  - assert write to srt with `out.srt` file and write to webvtt with `out.vtt`
- remove `TestWebVTTEscape` in `webvtt_test.go`

Reply:
Done. Test reflects the current `astistub.amp` implementation, but can be updated once that discussion is resolved.